src/wallet/rpcwallet.cpp

```diff
@@ -2409,6 +2493,10 @@ UniValue listunspent(const UniValue& params, bool fHelp)
             "      \"address\"   (string) Zcash address\n"
             "      ,...\n"
             "    ]\n"
+            "4. asOfHeight       (numeric, optional) Execute the query as if it were run when\n"
+            "                    the blockchain was at the height specified by this argument.\n"
+            "                    The default is to use the entire blockchain that the node is\n"
+            "                    aware of.\n"
```

Comment:
Document what happens if `asOfHeight` is greater than the height of the last known block.